backend/infrahub/config.py

```diff
@@ -134,7 +134,7 @@ class StorageSettings(BaseSettings):
 class DatabaseSettings(BaseSettings):
     model_config = SettingsConfigDict(env_prefix="INFRAHUB_DB_")
     db_type: DatabaseType = Field(
-        default=DatabaseType.MEMGRAPH, validation_alias=AliasChoices("INFRAHUB_DB_TYPE", "db_type")
+        default=DatabaseType.NEO4J, validation_alias=AliasChoices("INFRAHUB_DB_TYPE", "db_type")
```

Comment:
It doesn't have to be this PR but I think it would be nice to later get rid of the db_type and validation alias and rename db_type -> type.

Reply:
Yes but I'll let that for a next PR as you mentioned